amcl/src/amcl/sensors/amcl_laser.cpp

```diff
@@ -309,10 +315,13 @@ double AMCLLaser::LikelihoodFieldModel(AMCLLaserData *data, pf_sample_set_t* set
     total_weight += sample->weight;
   }
 
+  ROS_WARN("Invalid count %d", invalid_count);
```

Comment:
I think this should be debug or trace.  It happens a lot during normal operation.